### ISSUE TITLE
How to use legacy tag with lazy.nvim ?

### ISSUE THREAD

**Issue by MontiL** [j-hui/fidget.nvim#133]
my config in nvim/lua/plugins/lsp.lua:

 ```
 {
    "j-hui/fidget.nvim",
    config = function() require("fidget").setup() end,
    dependencies = { "neovim/nvim-lspconfig" }
  },
```
Not sure where to add this tag 

Thank you in advance for your support

**Comment by vhristev** [comment]
Here are some examples

Yours 
```
use({
  "j-hui/fidget.nvim",
  config = function()
    require("fidget").setup()
  end,
  tag = "legacy",
  requires = { "neovim/nvim-lspconfig" }
})
```

Mine
```
	use({ -- LSP Configuration & Plugins
		"neovim/nvim-lspconfig",
		requires = {
			-- Automatically install LSPs to stdpath for neovim
			"williamboman/mason.nvim",
			"williamboman/mason-lspconfig.nvim",
			"williamboman/nvim-lsp-installer",
			"jose-elias-alvarez/null-ls.nvim",
			-- Useful status updates for LSP
			{ 'j-hui/fidget.nvim', tag = 'legacy' },
			-- Additional lua configuration, makes nvim stuff amazing
			"folke/neodev.nvim",
		},
	})

```

Single line
`{ 'j-hui/fidget.nvim', tag = 'legacy' },`

**Comment by qff233** [comment]
> Here are some examples
> 
> Yours
> 
> ```
> use({
>   "j-hui/fidget.nvim",
>   config = function()
>     require("fidget").setup()
>   end,
>   tag = "legacy",
>   requires = { "neovim/nvim-lspconfig" }
> })
> ```
> 
> Mine
> 
> ```
> 	use({ -- LSP Configuration & Plugins
> 		"neovim/nvim-lspconfig",
> 		requires = {
> 			-- Automatically install LSPs to stdpath for neovim
> 			"williamboman/mason.nvim",
> 			"williamboman/mason-lspconfig.nvim",
> 			"williamboman/nvim-lsp-installer",
> 			"jose-elias-alvarez/null-ls.nvim",
> 			-- Useful status updates for LSP
> 			{ 'j-hui/fidget.nvim', tag = 'legacy' },
> 			-- Additional lua configuration, makes nvim stuff amazing
> 			"folke/neodev.nvim",
> 		},
> 	})
> ```
> 
> Single line `{ 'j-hui/fidget.nvim', tag = 'legacy' },`

hey, your example is used for packer, but he use lazy.nvim!

**Comment by qff233** [comment]
``` lua
	{
		"j-hui/fidget.nvim",
		requires = { "neovim/nvim-lspconfig" },
		tag = "legacy",
		config = true,
	},
```

here is an example. After you change the config, lauch :Lazy to update. It will change branch on legacy whose tag is legacy.

**Comment by wookayin** [comment]
@j-hui I don't think the use of `legacy` branch and therefore asking all the users to change their config was a good choice.

If you intended a breaking change, you instead could have made a stable tag/release with the current working (stable) version while working on experimental features on a new, non-default branch or the default branch (if a 'release' is used).

**Comment by j-hui** [comment]
Closed by #134

**Comment by jamietanna** [comment]
I appear to still be seeing this with the following config:

```lua
	{
		"j-hui/fidget.nvim",
		tag = "legacy",
		opts = {},
	},
```

This is resolved by:

```diff
 	{
 		"j-hui/fidget.nvim",
 		tag = "legacy",
- 		opts = {},
 	},
```
